src/nearby-stop-places/NearbyStopPlacesScreenComponent.tsx

```diff
@@ -167,6 +162,7 @@ export const NearbyStopPlacesScreenComponent = ({
         // Quick fix for iOS to fix stuck spinner by removing the RefreshControl when not focused
         isFocused || Platform.OS === 'android' ? (
           <RefreshControl
+            progressViewOffset={90}
```

Comment:
What was the issue here, without the offset?

Reply:
Hmm, I checked again and it's weird, without the offset, the refresh circle doesn't show up properly on dev builds, but in Prod it works properly.

Reply:
It looks like this without the offset

https://github.com/user-attachments/assets/26e01c90-6995-490a-9a90-d2e220572868

Reply:
Reverting this change